stacks/apps/augury.yml

```diff
@@ -300,7 +300,37 @@ Resources:
         - { Key: prx:dev:family, Value: Dovetail }
         - { Key: prx:dev:application, Value: Augury }
       TaskRoleArn: !GetAtt TaskRole.Arn
-
+  UploadHttpSigningUser:
```

Comment:
I would probably change this resource logical ID (and the others) to something like `S3SigningUser` or similar. This name came from when it was part of `upload.prx.org` and part of an HTTP API.